packages/pilot/src/models/dateSelectorPresets.js

```diff
@@ -1,27 +1,56 @@
-export default [
+export default t => [
```

Comment:
transformar em uma função nomeada ou atribuir a função a uma const apenas pra facilitar a visualizacão da stacktrace em algum problema.
```suggestion
const dateSelectorPresets = t => { .... }
export default dateSelectorPresets
```

ou 

```suggestion
export default function dateSelectorPresets (t) { .... }
```